src/stable/f5-bigip-ctlr/README.md

```diff
@@ -0,0 +1,53 @@
+# Helm Chart for the F5 BIG-IP Controller
+
+This chart simplifies repeatable, versioned deployment of the [F5 BIG-IP Controller for Kubernetes](http://clouddocs.f5.com/products/connectors/k8s-bigip-ctlr/latest/).
+
+## Requirements
+
+The chart presumes:
+- You have added a BIG-IP device to either your [Kubernetes](http://clouddocs.f5.com/containers/latest/kubernetes/kctlr-use-bigip-k8s.html) or [OpenShift](http://clouddocs.f5.com/containers/v2/openshift/kctlr-use-bigip-openshift.html) cluster.
+- A partition (other than common) exists on your BIG-IP for the instance of the controller to manage.
+- A secret exists in your cluster to access the BIG-IP.
+
+The default values for the partition and the secret to access the BIG-IP are:
+- `f5-bigip-ctlr` and 
+- `f5-bigip-ctlr-login` 
+
+These values can be easily updated using either `--set <param>=<value>` or `-f <values-file.yaml>`. See [customizing the chart before installing](https://docs.helm.sh/using_helm/#customizing-the-chart-before-installing) for more details.
+
+## Chart Details
+
+The chart creates a Deployment for one Pod containing the [k8s-bigip-ctlr](http://clouddocs.f5.com/products/connectors/k8s-bigip-ctlr/latest/) and the supporting RBAC resources.
+
+## Installing the Chart
+
+If using the names in the default values;
```

Comment:
Run the command shown below to install the chart using the default values.

Reply:
Done.